modules/message-lane/src/weights.rs

```diff
@@ -72,100 +73,105 @@ pub trait WeightInfo {
 pub struct RialtoWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for RialtoWeight<T> {
 	fn send_minimal_message_worst_case() -> Weight {
-		(138_421_000 as Weight)
+		(140_645_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(12 as Weight))
 	}
 	fn send_1_kb_message_worst_case() -> Weight {
-		(142_633_000 as Weight)
+		(146_434_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(12 as Weight))
 	}
 	fn send_16_kb_message_worst_case() -> Weight {
-		(194_483_000 as Weight)
+		(214_721_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(12 as Weight))
 	}
+	fn increase_message_fee() -> Weight {
+		(8_395_221_000 as Weight)
```

Comment:
The fee is quite large because we're reading the maximal possible message payload (`Vec<u8>`) in the benchmark. We can't actually predict this size without reading => can't use user-provided params.